poc/vdaf_prio3.py

```diff
@@ -483,6 +510,51 @@ class Prio3HistogramWithLength(Prio3Histogram):
         return Prio3HistogramWithLength
 
 
+class Prio3SumVecWithMultiproof(Prio3SumVec):
+    ID = 0xFFFFFFFF  # To be set by `with_params`
```

Comment:
Why not just set it here? If this behavior is needed, then don't set the ID here.